paparazzi-gradle-plugin/src/test/projects/compose/src/test/java/app/cash/paparazzi/plugin/test/LoggerLeakTest.kt

```diff
@@ -0,0 +1,49 @@
+package app.cash.paparazzi.plugin.test
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.layout.SubcomposeLayout
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+import org.junit.runners.model.Statement
+
+class LoggerLeakTest {
+  private val paparazzi: Paparazzi = Paparazzi()
+  private val expectExceptionRule: TestRule = TestRule { base, _ ->
+    object : Statement() {
+      override fun evaluate() {
+        var exception: Exception? = null
+        try {
+          base?.evaluate()
+        } catch (e: Exception) {
+          exception = e
+        }
+        assert(exception != null)
+      }
+    }
+  }
+
+  @get:Rule val ignored: RuleChain = RuleChain.outerRule(expectExceptionRule).around(paparazzi)
+
+  @Test
+  fun test1() {
+    paparazzi.snapshot { ComposeContent() }
+  }
+
+  @Test
+  fun test2() {
+    paparazzi.snapshot { ComposeContent() }
+  }
+
+  @Test
+  fun test3() {
+    paparazzi.snapshot { ComposeContent() }
+  }
```

Comment:
nit: why 3 tests?  would a single or two demonstrate the before/after as well?

Reply:
Will change to just 2